src/main/java/racingcar/car/Car.java

```diff
@@ -0,0 +1,27 @@
+package racingcar.car;
+
+import racingcar.engine.CarEngine;
+
+public class Car {
+    public static final int INITIAL_LOCATION = 0;
+
+    private Position position;
+
+    private Car() {
+        this.position = new Position(INITIAL_LOCATION);
+    }
+
+    public static Car createCar() {
+        return new Car();
+    }
+
+    public void move(CarEngine carEngine) {
+        if (carEngine.hasEnoughPower()) {
+            position.add();
```

Comment:
```suggestion
            this.position = position.add();
```

값객체로 새롭게 할당된걸 치환해줘야 원하는대로 동작할것 같아요!